modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc

```diff
@@ -75,7 +76,7 @@ class NDB_Menu_Filter_Form_mri_violations extends NDB_Menu_Filter_Form
             // if selection<>0, then insert new row.
             else{
                 // no need to insert to DB for Unresolved value.
-                if($val=='unresolved'){
+                if($val=='unresolved') {
```

Comment:
Are these changes to please PHPCS or just because the lack of a space was annoying you? Because this would still fail PHPCS since there's no space between "if" and "("..